app/grandchallenge/algorithms/models.py

```diff
@@ -103,6 +103,11 @@ class Algorithm(UUIDModel, TitleSlugDescriptionModel, ViewContentMixin):
         blank=True,
         on_delete=models.SET_NULL,
     )
+    optional_hanging_protocols = models.ManyToManyField(
```

Comment:
You need to define an explicit through table for new ManyToManyFields as this helps with long term maintenance https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.ManyToManyField.through

Same comment for the other 3 apps.

Reply:
just testing this now, but think I've got it

Reply:
I also see why it's best practice, too - you can't alter field to switch to a through table, you have to drop and re-add the whole thing.

Reply:
I think this is right, as per the latest commit